js/utils.js

```diff
@@ -340,6 +340,66 @@ Fliplet.Registry.set('dynamicListUtils', (function () {
     return record.indexOf(value) > -1;
   }
 
+  function filterDOMItem(filter) {
+    var $filter = $(filter);
+  
+    return '<div class="btn hidden-filter-controls-filter mixitup-control-active"'
+      + 'data-toggle="'+ $filter.data('toggle') +'"'
+      + ' data-field="'+ $filter.data('field') +'"'
+      + ' data-value="'+ $filter.data('value') +'"'
+      + '>'+ $filter.data('value')
+      + '</div>';
+  }
+
+  function transferActiveFilterClick(event) {
+    var $target = $(this);
+    var context = event.data.context;
+    var filterOverlay = event.data.filterOverlayClass;
+    var redirectSelector = context.data.filtersInOverlay 
+      ? filterOverlay + ' [data-filter-group] .mixitup-control-active[data-value="'+ $target.data('value') +'"]'
+      : '.hidden-filter-controls [data-filter-group] .mixitup-control-active[data-value="'+ $target.data('value') +'"]';
+    var $redirectTarget = context.$container.find(redirectSelector);
+  
+    $redirectTarget.trigger('click');
+  
+    if (context.data.filtersInOverlay) {
+      var $applyBtn = context.$container.find('.filter-header-holder .filter-header-btn-controls .apply-filters');
+  
+      // Allow UI update before we click apply button
+      setTimeout(function() {
+        $applyBtn.trigger('click');
+      }, 0)
+    }
+  }
+
+  function addActiveFilters(options) {
+    if (!options.filtersInOverlay) {
+      return;
+    }
+
+    var $container = options.context.$container;
+    var $selectedFilters = $container.find('[data-filter-group] .mixitup-control-active');
+    var $activeFiltersHolder = $container.find('.active-filters');
+    var $filtersGroup = $activeFiltersHolder.find('[data-filter-active-group]');
+    var activeFiltersElemenst = [];
+  
+    if (!$selectedFilters.length) {
+      $activeFiltersHolder.addClass('hidden');
+      return;
+    }
+  
+    $filtersGroup.html('');
+    $selectedFilters.each(function() {
+      var $activeFilter = $(filterDOMItem(this));
```

Comment:
Use `this` to determine the values, e.g. `this.dataset.toggle`, so that `filterDOMItem()` does not need to wrap `this` with another `$()` which slows it down.

Also, the `$()` wrapper for `filterDOMItem()` is also unnecessary if you use event delegation to handle the filter click events.

```suggestion
      options = { toggle, field, value };

      var activeFilter = filterDOMItem(options);
```